keps/sig-node/1029-ephemeral-storage-quotas/README.md

```diff
@@ -709,11 +734,161 @@ and are not reported here.
 | du after umount/mount    | 66.0      | 82.4  | 29.2         | 28.1   |
 | Remove Files             | 188.6     | 156.6 | 90.4         | 81.8   |
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: LocalStorageCapacityIsolationFSQuotaMonitoring
+  - Components depending on the feature gate: kubelet
+
+This feature uses project quotas to monitor emptyDir volume storage consumption
+rather than filesystem walk for better performance and accuracy.
+
+###### Does enabling the feature change any default behavior?
+
+None. Behavior will not change. The change is the way to monitoring the volume
+like ephemeral storage volumes and emptyDirs.
+When LocalStorageCapacityIsolation is enabled for local ephemeral storage and the
+backing filesystem for emptyDir volumes supports project quotas and they are enabled,
+use project quotas to monitor emptyDir volume storage consumption rather than
+filesystem walk for better performance and accuracy.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, but only for newly created pods.
+- Existed Pods: If the pod was created with enforcing quota, disable the feature gate
+  will not change the running pod.
+- Newly Created Pods: After setting the feature gate to false, the newly created pod
+  will not use the enforcing quota.
```

Comment:
This isn't entirely clear to me. If a pod is created with it "on", then the monitoring is done using the quota, even if the kubelet is restarted with the feature gate "off"? So kubelet looks and doesn't do the `du` business for those with this enabled? Does the enforcement (i.e., eviction) still work for pods created with the quota method?

Reply:
I made a mistake here. 

After we disable the feature gate, pod quota monitoring will turn back to use `du`.

https://github.com/kubernetes/kubernetes/blob/1367cca8fd67b09606b01c0a9e46cef59aef3424/pkg/volume/util/fsquota/quota_linux.go#L269-L301
`SupportsQuotas()` will check the feature gate at first.

Updated.